keg_elements/crypto.py

```diff
@@ -0,0 +1,47 @@
+import os
+
+import cryptography.hazmat.primitives.ciphers as ciphers
+from cryptography.hazmat.backends import default_backend
+
+
+def aes_cipher(key, iv, mode=None):
```

Comment:
We need to be able to supply the IV when creating a cipher for decryption. Instead I enforced secure IV creation in the `aes_encryptor` function